source/samples/OpenGLCore/OpenGLCoreSample.cpp

```diff
@@ -95,7 +109,7 @@ int main()
     canvas->SetBackgroundColor(Gwk::Color(150, 170, 170, 255));
 
     // Create our unittest control (which is a Window with controls in it)
-    auto unit = new TestFrame(canvas);
+    auto unit = new Gwk::Test::TestFrame(canvas);
```

Comment:
This might have been my compiler settings causing this, but my compiler didn't like the namespaces not being there.